src/Libraries/PythonNodeModels/PythonNode.cs

```diff
@@ -29,13 +32,15 @@ public PythonCodeMigrationEventArgs(string oldCode, string newCode)
 
     public abstract class PythonNodeBase : VariableInputNode
     {
-        private PythonEngineVersion engine = PythonEngineVersion.IronPython2;
+        private PythonEngineVersion? engine = null;
```

Comment:
The engine became nullable, as it cannot be initialized at this point. We have no access to `DynamoModel`. Luckily JSON serialization handles this change without problems.